controlpanel/api/rules.py

```diff
@@ -145,6 +145,12 @@ def is_self(user, other):
 add_perm('api.reset_mfa', is_authenticated & is_superuser)
 
 
+add_perm('api.list_release', is_authenticated & is_superuser)
+add_perm('api.create_release', is_authenticated & is_superuser)
+add_perm('api.destroy_release', is_authenticated & is_superuser)
+add_perm('api.update_release', is_authenticated & is_superuser)
```

Comment:
I think there is some ambiguity here which we can remove by adding the word "tool" to the mix.

Users currently don't release webapps via the Control Panel but being more explicit here doesn't harm as it helps avoid any possible confusion.

I see below the form for example is called `ToolReleaseForm` which is an **excellent and very clear** name, so consider using something similar for the permissions name as well.

Reply:
We're configuring *releases* of Tools (i.e. a tool is Jupyter or RStudio). The name of the model is wrong IMHO :-P the `Tool` model should be called something like `ToolRelease`. :+1: But I felt that was a refactor too far. ;-)

Reply:
Having said that.... I'm not really fussed what we call things, but the way folks have talked about this work in the past is in terms of "releases" rather than "tools" (hence my following this convention). Basically, we've just gotta agree names... ;-)

Reply:
Agreed on agreeing names. 

These are permissions to list tool releases, create a tool release, etc...hence my suggestion. 
Within our AP domain there are also webapp releases. Users currently don't manage these in the Control Panel, but I still think it's less confusing to call the permissions above something like `api.list_tool_release`, `api.create_tool_release`, etc...as sidestep the potential ambiguity and issue.

Reply:
Ahh... I see what you mean now. :+1: on that then. ;-)

Reply:
Sorry must have explained myself really badly in the first comment 🤦

Reply:
Fixed